docs/networks/node-ops/access-onchain-data/evm-gateway/evm-gateway-setup.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Setting up an EVM Gateway node
+sidebar_label: EVM Gateway Setup
+sidebar_position: 3
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This guide is for running the [EVM Gateway](https://github.com/onflow/flow-evm-gateway) node on Flow. The EVM Gateway implements the
+[Ethereum JSON-RPC specification](https://ethereum.org/en/developers/docs/apis/json-rpc/) and is the only node type which accepts EVM 
+client connections. 
+
+The EVM Gateway is an unstaked, non-participating edge node which consumes Flow protocol state from the configured Flow Access Node 
```

Comment:
why do you say its `non-participating`? IMO, it participates by sending transactions. It doesn't participate in block production directly but does so indirectly.

Reply:
I'll correct it to say it isn't a participant in block production, would that work?